src/main/scala/ScalaModulePlugin.scala

```diff
@@ -147,6 +147,17 @@ object ScalaModulePlugin extends AutoPlugin {
 
   lazy val scalaModuleSettingsJVM: Seq[Setting[_]] = scalaModuleOsgiSettings
 
+  lazy val scalaModuleSettingsNative: Seq[Settings[_]] = Seq(
+    scalaVersion := "2.11.11",
+    skip in compile := System.getProperty("java.version").startsWith("1.6"),
+    test := {},
+    libraryDependencies := {
+      if (!scalaVersion.value.startsWith("2.11"))
+        libraryDependencies.value.filterNot(_.organization == "org.scala-native")
+      else libraryDependencies.value
+    }
```

Comment:
I find this extremely weird. You're trying to bypass Native compilation if the specified `scalaVersion` is not compatible with Scala Native. But that, in itself, is a workaround for the first problem: the project *should not even try* to `update/compile/test` for such a Scala version. The build matrix should be configured so that the Native projects are only built on the appropriate versions.